apps/site/util/hexToRGBA.ts

```diff
@@ -1,4 +1,4 @@
-export const hexToRGBA = (hex: string = '', alpha = 0.9) => {
+export const hexToRGBA = (hex = '', alpha = 0.9) => {
```

Comment:
Not strictly necessary for this PR, but I figured I'd throw it in 😄. This was a violation of [`@typescript-eslint/no-inferrable-types`](https://typescript-eslint.io/rules/no-inferrable-types). That's enabled in the _stylistic_ config, which isn't enabled in `main` or this PR.